Packs/ServiceNow/Integrations/ServiceNowv2/README.md

```diff
@@ -38,9 +38,7 @@ These scripts are wrapped around the incident table, so to wrap them around anot
    2. Under **Classifier**, select ServiceNow Classifier.
    3. Under **Mapper (incoming)**, select ServiceNow - Incoming Mapper.
    4. Under **Mapper (outgoing)**, select ServiceNow - Outgoing Mapper.
-   5. To enable mirroring to close a ticket in ServiceNow, select the **Close Mirrored XSOAR Incident** checkbox.
-
-        ![image](https://raw.githubusercontent.com/demisto/content/75395ba6d9118bc3a5a399a31d95de4dc27f0911/Packs/ServiceNow/Integrations/ServiceNowv2/doc_files/closing-mirror-xsoar.png)
+   5. To enable mirroring to close a ticket in ServiceNow, under the **Mirrored XSOAR Ticket closure method** dropdown, select the ticket closing method,
```

Comment:
I'm sorry, shouldn't this one numbers 5 and 6 have the description revered? 
To enable mirroring to close incidents in Xsoar its what you've added
And the second one is closing the ticket (meaning mirror out)?

Reply:
You are correct, i've reversed the descriptions.